tmp/build/patch-charts.sh

```diff
@@ -157,6 +157,9 @@ function patchTemplates() {
           -e 's/tag:.*$/tag: '${MAISTRA_VERSION}'/' \
           -e 's/image: *proxy_init/image: proxy-init-ubi8/' \
           -e 's/image: *proxyv2/image: proxyv2-ubi8/' ${HELM_DIR}/istio/values.yaml ${HELM_DIR}/istio-init/values.yaml
+
+		# TODO: currently we're appending. When this comes down from upstream, we'll want to replace this with a substitution
+		sed -i -e "\$aimage: prometheus-ubi8" ${HELM_DIR}/istio/charts/prometheus/values.yaml
```

Comment:
perhaps append this at the top of values.yaml, after "tag" perhaps? Just so it's more visible?